src/types/timezone.jl

```diff
@@ -43,13 +43,13 @@ TimeZone(::AbstractString, ::Class)
 function TimeZone(str::AbstractString, mask::Class=Class(:DEFAULT))
```

Comment:
Let's avoid having to to the conversion multiple times:
```suggestion
TimeZone(str::AbstractString, args...) = TimeZone(String(str), args...)

function TimeZone(str::String, mask::Class=Class(:DEFAULT))
```